app/controllers/users_controller.rb

```diff
@@ -88,7 +88,15 @@ def csv_import
       if anthology_id.present?
         anthology = Anthology.find(anthology_id)
         for user in invited_users do
-          unless anthology.users.include? user
+          unless user.rep_group_list.include?(anthology.slug)
```

Comment:
Is this definitely called during the invitation process?

Reply:
Yes - I tested by inviting new users and checking their `rep_group_list` property in the Rails console.